src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -8424,6 +8424,38 @@ def test_vmss_cross_tenant_update(self, resource_group, another_resource_group):
         ])
 
 
+class VMCreateFromACGToOtherSubScenarioTest(LiveScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_from_acg_image_to_other_sub')
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_from_acg_image_to_other_sub',
+                           parameter_name='another_resource_group', subscription=AUX_TENANT_INFRASTRUCTURE)
```

Comment:
```suggestion
                           parameter_name='another_resource_group', subscription=AUX_SUBSCRIPTION)
```
Please use the `AUX_SUBSCRIPTION` directly